src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.html

```diff
@@ -5,10 +5,144 @@ <h2>{{ tutorialGroup.title }}</h2>
+                        {{ tutorialGroup.capacity }}
+                    </div>
+
+                    <div>
+                        <span class="text-secondary" jhiTranslate="artemisApp.entities.tutorialGroup.registrations"></span><span class="text-secondary">:</span>
+                        {{ !tutorialGroup.numberOfRegisteredUsers || tutorialGroup.numberOfRegisteredUsers === 0 ? '-' : tutorialGroup.numberOfRegisteredUsers }}
+                    </div>
+
+                    <div class="mt-2">
+                        <span class="text-secondary" jhiTranslate="artemisApp.entities.tutorialGroup.utilization"></span><span class="text-secondary">:</span>
+                        <fa-icon class="text-secondary" [icon]="faQuestionCircle" ngbTooltip="{{ 'artemisApp.entities.tutorialGroup.utilizationHelpDetail' | artemisTranslate }}" />
+
+                        @if (utilization) {
+                            <div class="progress" role="progressbar" [attr.aria-valuenow]="utilization" aria-valuemin="0" aria-valuemax="100">
+                                <div
+                                    class="progress-bar"
+                                    [ngClass]="{ 'bg-warning': utilization >= 60 && utilization < 90, 'bg-danger': utilization >= 90 }"
+                                    [ngStyle]="{ width: utilization + '%' }"
+                                >
+                                    {{ utilization }}%
+                                </div>
+                            </div>
+                        } @else {
+                            <div class="progress" role="progressbar" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100">
+                                <div class="progress-bar bg-secondary" style="width: 100%">-</div>
+                            </div>
+                        }
+                    </div>
+                </div>
+            </jhi-icon-card>
+        </div>
+
+        <div class="col-12 col-lg-4 mt-3 mt-lg-0">
+            <jhi-icon-card [headerIcon]="faCircleInfo" headline="artemisApp.entities.tutorialGroup.information">
+                <div>
+                    <div>
+                        <span class="text-secondary" jhiTranslate="artemisApp.entities.tutorialGroup.language"></span><span class="text-secondary">:</span>
+                        {{ tutorialGroup.language }}
+                    </div>
+
+                    <div>
+                        <span class="text-secondary" jhiTranslate="artemisApp.entities.tutorialGroup.campus"></span><span class="text-secondary">:</span>
+                        {{ tutorialGroup.campus }}
+                    </div>
+
+                    <div>
+                        <span class="text-secondary" jhiTranslate="artemisApp.entities.tutorialGroup.schedule"></span><span class="text-secondary">:</span>
+                        {{ getTutorialTimeSlotString() }}
```

Comment:
Is there a way to pass an attribute instead of a function here? We want to avoid function calls in the templates as much as possible

Reply:
Good call, changed it [here].(https://github.com/ls1intum/Artemis/pull/9445/commits/0de2b91ab2cda99670a2a48544bae7d0d3d46992). 😊